Application/EdFi.Ods.SandboxAdmin.Web/EdFi.Ods.SandboxAdmin.Web.csproj

```diff
@@ -292,6 +293,9 @@
       <Name>EdFi.Ods.Common</Name>
     </ProjectReference>
   </ItemGroup>
+ <ItemGroup>
```

Comment:
fix the formatting please

Reply:
In fact you can include it into the item group below

Reply:
Fixed it . Thanks